src/main/rx/fport.c

```diff
@@ -169,14 +172,15 @@ static void fportDataReceive(uint16_t c, void *data)
     static timeUs_t lastFrameReceivedUs = 0;
     static bool telemetryFrame = false;
 
-    const timeUs_t currentTimeUs = micros();
+    const timeUs_t currentTimeUs = microsISR();
 
     clearToSend = false;
 
     if (framePosition > 1 && cmpTimeUs(currentTimeUs, frameStartAt) > FPORT_TIME_NEEDED_PER_FRAME_US + 500) {
         reportFrameError(DEBUG_FPORT_ERROR_TIMEOUT);
 
         framePosition = 0;
+        rxBuffer[rxBufferWriteIndex].frameStartTimeUs = 0;
```

Comment:
This seems to be wrong - there is no frame received here, and the time the last frame was received does not magically change to the startup of the firmware either.